src/chart.zoom.js

```diff
@@ -201,8 +201,13 @@ function panIndexScale(scale, delta, panOptions) {
 
 function panTimeScale(scale, delta, panOptions) {
 	var options = scale.options;
-	options.time.min = rangeMinLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) - delta));
-	options.time.max = rangeMaxLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.lastTick) - delta));
+	
+	var newMax = rangeMaxLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.lastTick) - delta));
+	var newMin = rangeMinLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) - delta));
+	if (newMax != options.time.max && newMin != options.time.min) {
```

Comment:
I tested your suggestion, but unfortunately that didn't work. It blocks panning to the left and has also the problem with the zoom effect, when hitting the right border.

Here are the cases that occur normally:

Case        | Max | Min|Update values?
----------|------|----|--------
pan left | newMax < oldMax | newMin < oldMin | yes
pan right | newMax > oldMax | newMin > oldMin | yes
pan left, rangeLimited | newMax < oldMax | newMin == oldMin | no
pan right, rangeLimited | newMax == oldMax | newMin > oldMin | no

The idea was to allow updating the values only in the first two cases.